src/modules/peek/Peek.UI/Extensions/WindowExtensions.cs

```diff
@@ -26,7 +26,7 @@ public static void BringToForeground(this Window window)
         {
             // Ability to be able to set the Window to the Foreground is very limited. A current workaround is simulating mouse input before bringing to the foreground.
             var windowHandle = window.GetWindowHandle();
-            Windows.Win32.UI.Input.KeyboardAndMouse.INPUT input = new Windows.Win32.UI.Input.KeyboardAndMouse.INPUT { type = Windows.Win32.UI.Input.KeyboardAndMouse.INPUT_TYPE.INPUT_MOUSE, Anonymous = { } };
+            Windows.Win32.UI.Input.KeyboardAndMouse.INPUT input = new() { type = Windows.Win32.UI.Input.KeyboardAndMouse.INPUT_TYPE.INPUT_MOUSE, Anonymous = { } };
```

Comment:
nit: if you want to simplify the code here, consider moving the  Windows.Win32.UI.Input as using.